clicommand/step_cancel.go

```diff
@@ -74,7 +74,7 @@ var StepCancelCommand = cli.Command{
 			Name:   "force-grace-period-seconds",
 			Value:  defaultCancelGracePeriod,
 			Usage:  "The number of seconds to wait for agents to finish uploading artifacts before transitioning unfinished jobs to a canceled state. ′--force′ must also be supplied for this to take affect",
-			EnvVar: "BUILDKITE_FORCE_GRACE_PERIOD_SECONDS,BUILDKITE_CANCEL_GRACE_PERIOD",
+			EnvVar: "BUILDKITE_FORCE_CANCEL_GRACE_PERIOD_SECONDS,BUILDKITE_CANCEL_GRACE_PERIOD",
```

Comment:
@DrJosh9000, @wolfeidau is it ok that this environment variable doesn't 1:1 match the flag? It feels a bit redundant to include the work "cancel" in the flag name - but fairly important to have it in the global ENV var name.

Reply:
We have some similar differences in other flags such as https://github.com/buildkite/agent/blob/main/clicommand/tool_sign.go#L123

Env vars are a global space so prefixing seems legit to me.

Reply:
I guess a legitimate prefix would technically be `BUILDKITE_STEP_CANCEL_FORCE_GRACE_PERIOD_SECONDS`.

Reply:
I think it's good - it's clear that this env var goes with this flag, and the flag belongs in the context of the command but the env var stands alone.